rero_ils/static/js/rero_ils/rero_keep_tab.js

```diff
@@ -0,0 +1,46 @@
+/*
+
+RERO ILS
+Copyright (C) 2020 RERO
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, version 3 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+(function ( $ ) {
+  $.fn.keeptab = function( options ) {
+    var opts = $.extend( {
+      tabsName: '.nav-tabs',
+      localeItemName: 'navkeeptab',
+      selector: '.nav-link'
+    }, options );
+    $(opts.tabsName).find(opts.selector).on('click', function() {
```

Comment:
I'm little bit confused with this notation without using `this`.
I think the code below should work too, but shouldn't ; isn't it ?
```
$('#dummy_id').keeptab({ localeItemName: 'document-tab'});
```
Why not simply use `$(this).find(opts.selector).on....` ?

Reply:
If you have two sections of tabs on same page, you can limit the search area for storage